File: maiworld-seller-interface/src/main/java/com/maiworld/seller/service/SealSchoolService.java
package com.maiworld.seller.service;

import java.util.List;

import com.maiworld.pojo.TbSealSchool;

import entity.PageResult;

public interface SealSchoolService {

	
	public List<TbSealSchool> findAll();
	
	public PageResult findPage(int pageNum, int pageSize);
	
	public void add(TbSealSchool sealSchool);
	
	public void update(TbSealSchool sealSchool);
	
	public TbSealSchool findOne(Long id);
	
	public void delete(Long [] ids);
	
	public PageResult findPage(TbSealSchool sealSchool , int pageNum, int pagrSize);
	
	
	
	
	
	
	
	
	
}

File: maiworld-seller-web/src/main/webapp/js/service/sealSchoolService.js
//服务层
		app.service('sealSchoolService' ,function($http){
    				this.findAll=function(){
    					return $http.get('../sealSchool/findAll.do');
    				}
    				
    				this.findPage=function(page, rows){
    					return $http.get('../sealSchool/findPage.do?page='+page+'&rows='+rows);
    				}
    				
    				this.findOne=function(id){
    					return $http.get('../sealSchool/findOne.do?id='+id);
    				}
    				
    				this.add=function(entity){
    					return $http.post('../sealSchool/add.do',entity);
    				}
    				
    				this.update=function(entity){
    					return $http.post('../sealSchool/update.do',entity);
    				}
    				
    				this.dele=function(ids){
    					return $http.get('../sealSchool/delete.do?ids='+ids);
    				}
    				
    				this.search=function(page, rows, searchEntity){
    					return $http.post('../sealSchool/search.do?page='+page+'&rows='+rows, searchEntity);
    				}
    			});